contacts/index.html

```diff
@@ -2,121 +2,194 @@
+                <div class="no-likes">
+                  <div class="number">0</div>
+                </div>
+                <button>-</button>
+                <img src="images/profile-pics/pic5.jfif" alt="pic1" class="avatar">
+                <input type="checkbox">
+                <span class="full-name">
+                  <span>Pope</span>
+                  <span>Philip</span>
+                </span>
+                <img src="images/trash-1.png" alt="trash" class="trash">
+              </div>
+            </li>
+
+            <li>
+              <div class="person-field" class="align-center">
+                <button>+</button>
+                <div class="no-likes">
+                  <div class="number">0</div>
+                </div>
+                <button>-</button>
+                <img src="images/profile-pics/pic6.jfif" alt="pic1" class="avatar">
+                <input type="checkbox">
+                <span class="full-name">
+                  <span>Viorel</span>
+                  <span>Popescu</span>
+                </span>
+                <img src="images/trash-1.png" alt="trash" class="trash">
+              </div>
+            </li>
+
+            <li>
+              <div class="person-field" class="align-center">
+                <button>+</button>
+                <div class="no-likes">
+                  <div class="number">0</div>
+                </div>
+                <button>-</button>
+                <img src="images/profile-pics/pic7.jfif" alt="pic1" class="avatar">
+                <input type="checkbox">
+                <span class="full-name">
+                  <span>Elon</span>
+                  <span>Musk</span>
+                </span>
+                <img src="images/trash-1.png" alt="trash" class="trash">
+              </div>
+            </li>
+
+
+          </ul>
+          <div class="align-center">
+            <button class="button">Delete All</button>
           </div>
-        </li>
-      </ul>
+        </div>
 
-      <h2>Favs</h2>
-      <ol>
-        <li>
-            work in progress
-        </li>
-        <li>
-            <p>work in progress</p>
-        </li>
-        <li>
-            <p>work in progress</p>
-        </li>
-      </ol>
+        <div class="favourites">
+          <ol  class="list">
+            <h2 class="align-center">Favourites</h2>
+            <li>
+                <p class="align-center">work in progress</p>
+            </li>
+            <li>
+                <p class="align-center">work in progress</p>
+            </li>
+            <li>
+                <p class="align-center">work in progress</p>
+            </li>
+          </ol>
+          <div class="align-center">
+            <button class="button">Clear</button>
+          </div>
+        </div>
+      </div>
 
-      <h3>Add new contact</h3>
-      <form>
+      <form class="add">
+        <h2 class="align-center">Add new contact</h2>
         <label>First name:</label>
```

Comment:
If you added a placeholder for each input, the label is not required anymore. You can remove them.